test/dateTime.js

```diff
@@ -15,6 +17,286 @@ describe('dateTime', () => {
 
 	afterEach(() => documentLocaleSettings.reset());
 
+	describe('converting date/time', () => {
+
+		function getExpectedResult(date, minutes, hours, offset) {
+			// calculate what expected hours and minutes should be based on offset
+			const offsetHours = Math.floor(Math.abs(offset));
+			if (offset < 0) {
+				hours -= offsetHours;
+			} else {
+				hours += offsetHours;
+			}
+			if ((offset % 1) !== 0) {
+				minutes = minutes + 60 * (offset % 1);
+				if (minutes > 60) {
+					minutes -= 60;
+					hours += 1;
+				}
+				hours = Math.floor(hours);
+			}
+
+			// adjust date based on number of hours
+			if (hours < 0) {
+				hours = 24 + hours;
+				date--;
+			} else if (hours >= 24) {
+				hours -= 24;
+				date++;
+			}
+
+			return {
+				date: date,
+				hours: hours,
+				minutes: minutes
+			};
+		}
+
+		describe('convertUTCToLocalDateTime', () => {
+			it('should return original date if timezone identifier is blank', () => {
+				const UTCDateTime = {
+					month: 8,
+					date: 25,
+					year: 2015,
+					hours: 12,
+					minutes: 28,
+					seconds: 0
+				};
```

Comment:
Seems like you're reusing the same date over and over again (which is fine), but you could maybe pull it out above and just reference it in your tests to make them smaller?